### PR TITLE
Unpin mermaid and specify which version of the JavaScript library to …

### PR DESCRIPTION
**Pull request by stevepiercy**
…load

From https://github.com/plone/plone-sphinx-theme/pull/22#issuecomment-2173067441

----
📚 Documentation preview 📚: https://plone-sphinx-theme--23.org.readthedocs.build/

### CODE REVIEW

File: requirements-docs.txt [hunk 1]
```diff
@@ -12,7 +12,7 @@ sphinx-reredirects
 sphinx-sitemap
 sphinxcontrib.httpdomain
 sphinxcontrib.httpexample
-sphinxcontrib-mermaid==0.8.1 # See https://github.com/mgaitan/sphinxcontrib-mermaid/issues/110#issuecomment-1473688354
+sphinxcontrib-mermaid
```
Comment (polyester) [inline requirements-docs.txt:15]: sphinxcontrib.mermaid

Comment (stevepiercy) [reply requirements-docs.txt:15]: Are you sure? See https://pypi.org/project/sphinxcontrib-mermaid/, plus using a dash works.

> ## Installation
> You can install it using pip
>
> `pip install sphinxcontrib-mermaid`

Comment (polyester) [reply requirements-docs.txt:15]: Not quite. Both pip install sphinxcontrib.mermaid and pip install sphinxcontrib-mermaid work somehow. However the docs also say to put sphinxcontrib.mermaid in the "extensions" list in conf.py

Comment (stevepiercy) [reply requirements-docs.txt:15]: The names of a PyPI package and its configuration in `conf.py` can differ, and often do. See https://github.com/plone/plone-sphinx-theme/pull/23/files#diff-85933aa74a2d66c3e4dcdf7a9ad8397f5a7971080d34ef1108296a7c6b69e7e3R74.

Comment (stevepiercy) [reply requirements-docs.txt:15]: Also in the install instructions in the docs:

> extensions = [
>     ...,
>     'sphinxcontrib.mermaid'
> ]

**Review by polyester — APPROVED** [review]

**Pull request merged**